src/main/kotlin/app/revanced/patches/youtube/layout/autorepeat/patch/AutoRepeatPatch.kt

```diff
@@ -0,0 +1,30 @@
+package app.revanced.patches.youtube.layout.autorepeat.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.impl.BytecodeData
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultError
+import app.revanced.patcher.patch.annotations.Dependencies
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.patch.impl.BytecodePatch
+import app.revanced.patches.youtube.layout.autorepeat.annotations.AutoRepeatCompatibility
+import app.revanced.patches.youtube.layout.autorepeat.fingerprints.AutoRepeatFingerprint
+import app.revanced.patches.youtube.misc.integrations.patch.IntegrationsPatch
+
+@Patch
+@Dependencies(dependencies = [IntegrationsPatch::class])
+@Name("disable-auto-repeat")
```

Comment:
Rename to hide-auto-repeat-button for consistency across our patch bundle

Reply:
but it does not hide a button. It disables auto repeat, theres no button for that

Reply:
Yeah I saw that now too. It's there in Vanced too. Trying to figure out what Vanced actually did with that lol

Reply:
Maybe "enable-autorepeat" is enough for the patch name?

Reply:
I think people will get confused by that

Reply:
Therefore a description is included. The name should be short and not responsible to completely descriptive. If you insist, maybe "autorepeat-by-default"